dateparser/date.py

```diff
@@ -228,6 +228,16 @@ def _is_valid_date_obj(self, date_obj):
 class DateDataParser(object):
 
     def __init__(self, languages=None, allow_redetect_language=False):
+        """
+        :param languages:
+                A list of two letters language codes.e.g. ['en', 'es'].
+                If languages are given, it will not attempt to detect the language.
+
+        :param allow_redetect_language:
+                Enables/disables language re-detection.
+        
+        :return: A parser instance
+        """
```

Comment:
I think it doesn't appear anywhere. Any suggestions, where to put this docstring if it makes sense using it?